src/lib/ThemeContext.tsx

```diff
@@ -101,6 +102,10 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({
     React.useState<ColorSchemeName>(colorSchemeFromRN)
 
   React.useEffect(() => {
+    if (isWeb) {
```

Comment:
Instead of doing this in an effect, can we just treat `colorSchemeFromRN` as the source of truth when `isWeb` is true? And ideally rewrite the ternary below to be easier to follow.